packages/wrangler/src/index.ts

```diff
@@ -204,33 +210,34 @@ export function createCLIParser(argv: string[]) {
 		})
 		.scriptName("wrangler")
 		.wrap(null)
+		.locale("en_US")
 		// Define global options here, so they get included in the `Argv` type of
 		// the `wrangler` variable
 		.version(false)
 		.option("v", {
-			describe: "Show version number",
+			describe: "⚑ Show version number",
 			alias: "version",
 			type: "boolean",
 		})
 		.option("config", {
 			alias: "c",
-			describe: "Path to .toml configuration file",
+			describe: "⚑ Path to .toml configuration file",
 			type: "string",
 			requiresArg: true,
 		})
 		.option("env", {
 			alias: "e",
-			describe: "Environment to use for operations and .env files",
+			describe: "⚑ Environment to use for operations and .env files",
```

Comment:
This emoji seems like visual clutter for not much benefit, especially since it's only on global flags (not command flags)

Reply:
I agree. Did that also come from the designs? Would be happy if this was not added.

Reply:
yeah, I 100% agree with you! I'm going to go ahead and break design here